CMakeLists.txt

```diff
@@ -21,7 +22,7 @@ function (set_flags target)
         target_compile_options(${target} 
             PRIVATE 
             /W4 
-            /WX
+#            /WX
```

Comment:
It's probably best we just get rid of this option entirely, not sure if anyone really likes warning being treated as errors 
```suggestion
```